sidecar-ffi/src/lib.rs

```diff
@@ -523,3 +527,118 @@ pub unsafe extern "C" fn ddog_sidecar_stats(
     buf.copy_from_slice(str.as_bytes());
     ffi::CharSlice::from_raw_parts(malloced as *mut c_char, size)
 }
+
+#[no_mangle]
+#[allow(clippy::missing_safety_doc)]
+pub unsafe extern "C" fn ddog_sidecar_dogstatsd_count(
+    transport: &mut Box<SidecarTransport>,
+    instance_id: &InstanceId,
+    metric: ffi::CharSlice,
+    value: u64,
+    tags: Option<&ddcommon_ffi::Vec<Tag>>,
```

Comment:
So we passa all the tags all the time? Some of them (e.g lang, lang_interpreter, lang_version, tracer_version, runtimeId won't change from one session to another. 
Also, not saying we should do it, but we do process at least some tags in .NET (cf [this](https://github.com/DataDog/dd-trace-dotnet/blob/bcc8488d70163ab8ec0bfb95163a58545874915b/tracer/src/Datadog.Trace/TracerManagerFactory.cs#L369)), do you do it on php side?

Reply:
Yes, we do that on the PHP side. The language/version/runtime id tags could be probably directly provided within the sidecar. PR #398 anyway transfers that language/version info as part of session info. We could just use that part and merge it here.
Would make sense to me, great point, Pierre!

Reply:
If processing happens on the client side we should document it Somewhere in the API.